docs/content/1.getting-started/2.authentication.md

```diff
@@ -105,3 +105,93 @@ watch(user, () => {
 ::tip
 If you want to manually set the redirect path, you can do so by disabling [`saveRedirectToCookie`](/getting-started/introduction#redirectoptions), and then set the value using the  [`useSupabaseCookieRedirect`](/composables/usesupabasecookieredirect) composable directly.
 ::
+
+### Reset Password
+
+You can easily handle password reset request for the user. The flow consist of 2 steps:
+
+1. Allow the user to login via the password reset link. When the user clicks the reset link in the email they are redirected back to your application.
```

Comment:
Could be visually nice to use https://ui.nuxt.com/getting-started/typography#steps